docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -198,14 +194,15 @@ processing within Iris should use four workers in a thread pool::
     These dask processing options will last for the lifetime of the Python session
     and must be re-applied in other or subsequent sessions.
 
-See the
+Other dask processing options are also available. See the
 `dask documentation <http://dask.pydata.org/en/latest/scheduler-overview.html>`_
 for more information on setting dask processing options.
 
 
 Further reading
 ---------------
 
-This section of the user guide has been designed to give a quick overview of the
-key concepts of real and lazy data within Iris. If you require more detail, we
-have produced a much more detailed whitepaper on the subject.
+This section of the user guide is intended as a quick overview of the
+key concepts of real and lazy data within Iris. For more detail and more in-depth
+discussions on the concepts introduced here and related concepts, see the related
+whitepaper.
```

Comment:
Yeah I should try and properly rewrite this little section, I've largely ignored it up until now in the hope that I wouldn't need to do anything with it.